cli/internal/run/dry_run.go

```diff
@@ -113,7 +113,7 @@ func populateCacheState(turboCache cache.Cache, taskSummaries []*runsummary.Task
 			for index := range queue {
 				task := taskSummaries[index]
 				itemStatus := turboCache.Exists(task.Hash)
-				task.CacheSummary = runsummary.NewTaskCacheSummary(itemStatus, nil)
```

Comment:
We lose this `nil` behavior. But now since now timeSaved is available everywhere (in theory), we don't need to distinguish from `0`.